tools/shoestring/shoestring/wizard/setup_file_generator.py

```diff
@@ -71,7 +74,7 @@ async def prepare_shoestring_config(network_type, config_filepath):
 		await run_init(InitArgs(network_type, config_filepath))
 
 
-async def prepare_shoestring_files(screens, directory):
+async def prepare_shoestring_files(screens, directory, shoestring_directory):  # pylint: disable=too-many-locals
```

Comment:
Do we still need `directory` as we can use `shoestring_directory` instead?

Reply:
Yes, the ``directory`` is a temp folder storing temp setup files.

Since import-bootstrap now stores import files in the config folder, the shoestring install folder is needed.